.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,46 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed in the form of fix #XX for auto-linking.
```

Comment:
I would map a closing keyword to the type of change, i.e. `[Fixes|Resolves] #000` (bug fixes) and `Closes #000`  (new feature)